.github/workflows/deploy.yml

```diff
@@ -0,0 +1,13 @@
+name: Deploy via single cell infra repo
+
+on:
+  push:
+    branches: master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: repository dispatch
+        run: |
+          curl -XPOST -u mdunitz:${{secrets.SCINFRA_TOKEN}} -H "Accept: application/vnd.github.everest-preview+json"  -H "Content-Type: application/json" https://api.github.com/repos/chanzuckerberg/single-cell-infra/dispatches --data '{"event_type": "cellxgene-hook"}'
```

Comment:
Is there a way to do this without tying it to you specific user name?  Maybe we invent a user for this purpose?

Reply:
I think creating a single-cell user makes sense. I'll talk to infra about doing that/create a ticket